ovgs.proto

```diff
@@ -1,175 +1,276 @@
 // Copyright (c) 2023 Arista Networks, Inc.
 // Use of this source code is governed by the BSD 2-Clause License
 // that can be found in the LICENSE file.
-
 syntax = "proto3";
 
+package ovgs.v1;
+
 import "google/protobuf/timestamp.proto";
 
-package ovgs.v1;
+option go_package = "github.com/aristanetworks/ownership-voucher-grpc/ovgs";
 
-option go_package = "arista/sztp/ovgs/gen";
+enum UserRole {
+  USER_ROLE_UNSPECIFIED = 0;
+  USER_ROLE_SUPPORT = 1;      // Internal to the service
```

Comment:
TODO: we'd like to split off ADMIN into ADMIN and ASSIGNER, where the latter does not have the ability to much with user roles. @harshitk-arista